ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -6,8 +6,9 @@
 
 
 def add_user(*args):
-    User = get_user_model()
-    User(serial='__system__', full_name='System', password='!!').save()
+    queryset = get_user_model().objects.get_queryset_unfiltered()
+    queryset.update_or_create(serial='__system__',
```

Comment:
I've been going back and forth on whether we should just create or `update_or_create` here.

At the point we run this migration, the `__system__` user really shouldn't exist, so the migration can't fail on the serial already being taken. It if does, it means somebody actually created a user with that serial, and I'm not sure we should silently change their password. :smile: 

However, failing in the migration means (at the moment) failing the Debian package update, which really must not happen, so I guess `update_or_create` is marginally better here.